package.json

```
@@ -2,6 +2,14 @@
   "name": "mocha",
   "version": "7.2.0",
   "description": "simple, flexible, fun test framework",
+  "main": "./index.js",
+  "exports": {
+    ".": {
+      "require": "./index.js",
+      "import": "./index.mjs"
+    },
+    "./": "./"
+  },
```

For maximum compat, including node v13.0 and v13.1:
```suggestion
  "exports": {
    ".": [
      {
        "require": "./index.js",
        "import": "./index.mjs",
        "default": "./index.js"
      },
      "./index.js"
    ],
    "./": "./"
  },
```

---

Done